test/e2e/libpod_suite_test.go

```diff
@@ -460,3 +460,36 @@ func (p *PodmanTest) BuildImage(dockerfile, imageName string) {
 	session.Wait(120)
 	Expect(session.ExitCode()).To(Equal(0))
 }
+
+//GetHostDistribution returns the dist in string format. If the
+//distribution cannot be determined, an empty string will be returned.
+//Possible returns must be RHEL, Ubuntu, CentOS, Fedora
+func (p *PodmanTest) GetHostDistribution() string {
```

Comment:
Nothing appears to use this right now?

Reply:
Nope, i was using it during test to skip a specific test for centos but was able to fix the test. I thought that it might be useful in the future.